post/static/css/authors.css

```diff
@@ -0,0 +1,50 @@
+.rounded-writer {
+	border-radius:2rem;
```

Comment:
sa indent
em check lại indent trong file này nhé, setting indent trong VS code như trc chị có hướng dẫn rồi
`Spaces -> Convert indentation to spaces`

Reply:
dạ